src/main/java/nl/codefox/gilmore/Gilmore.java

```diff
@@ -2,7 +2,6 @@
 
 import net.dv8tion.jda.JDA;
 import net.dv8tion.jda.JDABuilder;
-
```

Comment:
It would be helpful if you set up your imports ordering and style to be the same. If you use IntelliJ I can give you a settings file.
Now it just clutters other, meaningful changes with these random reorderings